src/misc.ts

```diff
@@ -633,3 +634,20 @@ const getAllRemovedRowsBeforeRowIndex = (rowIndex: number, removedRowsIndexs: nu
 
 export const getDatagridRowDataIndex = (rowIndex: number, removedRowsIndexs: number[]) =>
   rowIndex + getAllRemovedRowsBeforeRowIndex(rowIndex, removedRowsIndexs).length;
+
+export const fuzzySearch = <T>(array: T[], query: string | undefined, keys: string[]) => {
+  if (!query) {
+    return array;
+  }
+
+  const fuse = new Fuse(array, {
+    keys,
+    includeScore: true,
+    threshold: 0.3,
+  });
+
+  return fuse
+    .search(query.toLocaleLowerCase())
+    .filter(({ score }) => score !== 1)
```

Comment:
question: Previuse solution uses a threshold, don't we need to support it too?

Reply:
This solution also uses threshold of 0.3 - the results get rounded down so they can be up to 0.4 (0.39..) so in theory we don't need this filter here. I'll remove it